x-pack/plugins/security_solution/server/lib/detection_engine/routes/users/suggest_user_profiles_route.ts

```diff
@@ -22,14 +22,14 @@ export const suggestUserProfilesRoute = (
   router.versioned
     .get({
       path: DETECTION_ENGINE_ALERT_SUGGEST_USERS_URL,
-      access: 'public',
+      access: 'internal',
       options: {
         tags: ['access:securitySolution'],
       },
     })
     .addVersion(
       {
-        version: '2023-10-31',
+        version: '1',
```

Comment:
We also have an [openAPI schema](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/common/api/detection_engine/users/suggest_user_profiles_route.schema.yaml#L4) for this route; that should be updated as well 👍

Reply:
Thanks for catching that! Updating.